spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -369,14 +369,14 @@ class Advert < ActiveRecord::Base
 
         add_index :adverts, [:category_id], name: 'on_category_id'
 
-        #{"execute \"ALTER TABLE adverts ADD CONSTRAINT index_adverts_on_category_id FOREIGN KEY index_adverts_on_category_id(category_id) REFERENCES categories(id) \"\n" if defined?(Mysql2)}
+        #{"add_foreign_key(\"adverts\", \"categories\", name: \"index_adverts_on_category_id\")\n" if defined?(Mysql2)}
       EOS
       .and migrate_down(<<~EOS.strip)
         remove_column :adverts, :category_id
 
         remove_index :adverts, name: :on_category_id rescue ActiveRecord::StatementInvalid
 
-        #{"remove_foreign_key('adverts', name: '')\n" if defined?(Mysql2)}
+        #{"remove_foreign_key(\"adverts\", name: nil)\n" if defined?(Mysql2)}
```

Comment:
This change is surprising. Is `nil` allowed as the name? I would have expected `""` was right.